### PR TITLE
add a test

### PR DESCRIPTION
**Pull request by DmitriPanichin**

### CODE REVIEW

File: src/test/java/school/redrover/LocomotiveGroupTest.java [hunk 1]
```diff
@@ -138,4 +139,51 @@ public void yandexSearchBarTest() throws InterruptedException {
             driver.quit();
         }
       }
+
+    @Test
+
+    public void simpleSearchTest() throws InterruptedException {
+        WebDriver driver = new FirefoxDriver();
```
Comment (tereshenkov29) [inline src/test/java/school/redrover/LocomotiveGroupTest.java:146]: According to the 19th lecture, we should use Chrome browser only
Not sure is it related to the Jenkins only or to all tests but still

Comment (DmitriPanichin) [reply src/test/java/school/redrover/LocomotiveGroupTest.java:146]: On FireFox the test run more quick then on Chrome, but i could change it.

File: src/test/java/school/redrover/LocomotiveGroupTest.java [hunk 2]
```diff
@@ -138,4 +140,50 @@ public void yandexSearchBarTest() throws InterruptedException {
             driver.quit();
         }
       }
+    @Ignore
+    @Test
+    public void simpleSearchTest() throws InterruptedException {
```
Comment (tereshenkov29) [inline src/test/java/school/redrover/LocomotiveGroupTest.java:145]: it should be a "test" prefix - not simpleSearchTest but testSimpleSearch

Comment (DmitriPanichin) [reply src/test/java/school/redrover/LocomotiveGroupTest.java]: Done both changes

**Review by tereshenkov29 — APPROVED** [review]

**Review by Michael-Rublev — APPROVED** [review]

File: src/test/java/school/redrover/LocomotiveGroupTest.java [hunk 3]
```diff
@@ -138,4 +140,50 @@ public void yandexSearchBarTest() throws InterruptedException {
             driver.quit();
         }
       }
+    @Ignore
+    @Test
+    public void testSimpleSearch() throws InterruptedException {
+        WebDriver driver = new ChromeDriver();
```
Comment (EkaterinaSokolova-V) [inline src/test/java/school/redrover/LocomotiveGroupTest.java:146]: Please use getDriver() method (from BaseTest class) istead of your driver.

**Pull request merged**